.github/scripts/install_go.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -eo pipefail
+
+# install Go compiler
```

Comment:
What is go needed for? Is this file needed?

Reply:
I remove that script. Go is needed to install singularity but it is already installed in the GitHub runners.

Reply:
Wow, OK. Last time I looked, singularity was written in a different language. They re-wrote the whole thing in go!